notebooks/05_Kubeflow_Pipeline/05_05_Pipeline_EMR.ipynb

```diff
@@ -51,9 +51,13 @@
    "outputs": [],
    "source": [
     "# Create a bucket to store sample data\n",
-    "S3_BUCKET='kubeflow-pipeline-data'\n",
+    "HASH = 'abc123'\n",
```

Comment:
Can you try to find a way to generate a hash instead of write a hash here?

Reply:
I'll try use other method to generate hash for usage

Reply:
Fixed in the latest version